pkg/manager/member/utils.go

```diff
@@ -447,3 +448,55 @@ func shouldRecoverDM(dc *v1alpha1.DMCluster, component string, podLister corelis
 	}
 	return true
 }
+
+func CreateOrUpdateService(serviceLister corelisters.ServiceLister, serviceControl controller.ServiceControlInterface, newSvc *corev1.Service, monitor *v1alpha1.TidbMonitor) error {
```

Comment:
```suggestion
func CreateOrUpdateService(serviceLister corelisters.ServiceLister, serviceControl controller.ServiceControlInterface, newSvc *corev1.Service, obj runtime.Object) error {
```

Reply:
```
	meta, ok := obj.(metav1.Object)
	if !ok {
		return fmt.Errorf("%T is not a metav1.Object", obj)
	}
```

Reply:
Then we can use the same function in the `syncTiDBService` function in `tidb_member_manager.go`.